### PR TITLE
refactor(web): dns anycast modal

### PR DESCRIPTION
**Pull request by frenautvh**
Replace "DNS Anycast" activation modal with a single page

See : MANAGER-3461

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain/dns/domain-dns.controller.js [hunk 1]
```diff
@@ -5,223 +5,231 @@ import remove from 'lodash/remove';
...
-        table: false,
-      };
-      this.urls = {
-        zoneCheck: this.constants.urls.TOOLS.ZONE_CHECK,
-      };
-
-      this.$scope.$on('Domain.Dns.Reload', () => this.init());
-      this.$scope.loadTable = () => this.loadTable();
+export default class DomainDnsCtrl {
+  constructor(
```
Comment (cbourgois) [inline packages/manager/apps/web/client/app/domain/dns/domain-dns.controller.js]:
```suggestion
  /* @ngInject */
  constructor(
```

File: packages/manager/apps/web/client/app/domain/anycast/domain-dns-anycast.html [hunk 2]
```diff
@@ -1,45 +1,37 @@
-<div data-ng-controller="DomainDnsAnycastActivateCtrl as ctrl">
-    <div data-wizard
-         data-wizard-bread-crumb
-         data-wizard-confirm-button-text="'wizard_pay' | translate"
-         data-wizard-on-cancel="resetAction"
-         data-wizard-on-finish="displayBC"
-         data-wizard-title="'domain_configuration_dnsanycast_title' | translate">
+<div>
```
Comment (FredericEspiau) [inline packages/manager/apps/web/client/app/domain/anycast/domain-dns-anycast.html]: Could be removed

File: packages/manager/apps/web/client/app/domain/anycast/domain-dns-anycast.html [hunk 3]
```diff
@@ -1,45 +1,37 @@
...
+                 data-ng-if="$ctrl.error">
+        <p data-ng-bind="$ctrl.error"></p>
+    </oui-message>
 
-            <p data-ng-bind-html="'domain_configuration_dnsanycast_step2_question' | translate: { t0: ctrl.domainName }"></p>
+    <div data-ng-if="!$ctrl.loading">
+        <div data-ng-if="!$ctrl.order">
+            <p data-ng-bind-html="'domain_configuration_dnsanycast_step1_explanation' | translate"></p>
+            <p>
+                <span data-ng-bind-html="'domain_configuration_dnsanycast_step1_explanation2' | translate: { t0: '<strong>' + $ctrl.optionDetails.withoutTax + '</strong>' }"></span>
```
Comment (FredericEspiau) [inline packages/manager/apps/web/client/app/domain/anycast/domain-dns-anycast.html]: Could be in the `p`

File: packages/manager/apps/web/client/app/domain/domain.routes.js [hunk 4]
```diff
@@ -63,12 +66,15 @@ angular.module('App').config(($stateProvider) => {
...
       ],
+      goToDns: /* @ngInject */ $state => () => $state.go('app.domain.alldom.dns'),
+      goToDnsAnycast: /* @ngInject */ $state => () => $state.go('app.domain.alldom.anycast'),
     },
     translations: { value: ['../core', '../domain', '../email', '../hosting', '../domain-operation'], format: 'json' },
   });
 
   ['product', 'alldom'].forEach((stateType) => {
     // Clone state before using it as it will be modified by UI Router
+    $stateProvider.state(`app.domain.${stateType}.anycast`, clone(anycastState));
```
Comment (marie-j) [inline packages/manager/apps/web/client/app/domain/domain.routes.js]: Could `anycast` routing be declared in its submodule ?

**Pull request merged**